tests/std/tests/P0220R1_string_view/test.cpp

```diff
@@ -1088,11 +1088,30 @@ static_assert(u8"abc"sv[1] == u8'b');
 static_assert(noexcept(u8"abc"sv));
 #endif // __cpp_char8_t
 
+constexpr bool test_case_range_constructor() {
+#if _HAS_CXX23 && defined(__cpp_lib_concepts)
+    const array expectedData{'n', 'o', ' ', 'n', 'u', 'l', 'l'};
+    // Also tests the corresponding deduction guide:
+    same_as<string_view> auto sv = basic_string_view(expectedData);
+    assert(sv.data() == expectedData.data());
+    assert(sv.size() == 7);
+    assert(sv.length() == 7);
+    assert(!sv.empty());
+    assert(sv[1] == 'o');
+    assert(sv.at(1) == 'o');
+    assert(sv.front() == 'n');
+    assert(sv.back() == 'l');
+#endif // _HAS_CXX23 && defined(__cpp_lib_concepts)
+
+    return true;
+}
+

```

Comment:
We're missing a `static_assert(test_case_range_constructor());` here somewhere to ensure the new constructor works properly while evaluating a constant expression.